commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/portlet/action/EditCompareProductMVCActionCommand.java

```diff
@@ -59,10 +59,10 @@ protected void doProcessAction(
 		boolean compare = ParamUtil.getBoolean(actionRequest, compareParam);
 
 		if (compare) {
-			CPCompareUtil.addCompareProduct(httpServletRequest, cpDefinitionId);
+			CPCompareHelperUtil.addCompareProduct(httpServletRequest, cpDefinitionId);
```

Comment:
@alecsloan in an OSGi component like this, we should be using a `_cpCompareHelper` reference rather than calling the static util directly.